scripts/tests/chiptest/test_definition.py

```diff
@@ -408,21 +417,34 @@ def Run(self, runner, apps_register, paths: ApplicationPaths, pics_file: str,
 
             if test_runtime == TestRunTime.CHIP_REPL_PYTHON:
                 chip_repl_yaml_tester_cmd = paths.chip_repl_yaml_tester_cmd
-                python_cmd = chip_repl_yaml_tester_cmd + \
-                    ['--setup-code', setupCode] + ['--yaml-path', self.run_name] + ["--pics-file", pics_file]
+                python_cmd = (chip_repl_yaml_tester_cmd
+                              + ['--setup-code', setupCode]
+                              + ['--yaml-path', self.run_name]
+                              + ["--pics-file", pics_file])
                 if dry_run:
                     logging.info(" ".join(python_cmd))
                 else:
                     runner.RunSubprocess(python_cmd, name='CHIP_REPL_YAML_TESTER',
                                          dependencies=[apps_register], timeout_seconds=timeout_seconds)
             else:
-                pairing_cmd = paths.chip_tool_with_python_cmd + ['pairing', 'code', TEST_NODE_ID, setupCode]
+                pairing_server_args = []
+                if ble_controller_tool is not None:
+                    # Explicitly use ble-wifi pairing to force the use of BLE
+                    # exchange and AP association. The code-wifi pairing skips
+                    # BLE commissioning if devices can be commissioned over IP
+                    # network.
```

Comment:
Can we fix the server to not be advertising on the IP network?   It's _really_ weird (like: forbidden by the spec) to be advertising both on the IP network and over BLE.

Reply:
I've changed the test setup. Now, the IP on the veth link (which name is prefixed with `wl` so it is recognized as wlan) is assigned only after successful wifi acquisition (done by the mocked WPA supplicant). With that setup our Matter application is not linked with controller over veth, so the code-wifi pairing successfully selects BLE for commissioning.